app/model/repository/ExerciseTags.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Model\Repository;
+
+use App\Model\Entity\Exercise;
+use App\Model\Entity\ExerciseTag;
+use Kdyby\Doctrine\EntityManager;
+
+class ExerciseTags extends BaseRepository {
+
+  public function __construct(EntityManager $em) {
+    parent::__construct($em, ExerciseTag::class);
+  }
+
+  public function findByNameAndExercise(?string $name, Exercise $exercise): ?ExerciseTag {
```

Comment:
Is it really possible (wise) that `$name === null` in this context?

Reply:
It is not